server/sonar-db-migration/src/main/java/org/sonar/server/platform/db/migration/AutoDbMigration.java

```diff
@@ -75,6 +77,27 @@ void installH2() {
   @VisibleForTesting
   protected void createSchema(Connection connection, String dialectId) {
```

Comment:
yeah, it can now be renamed. Forgot to do that when I changed this class to run the migration engine